### PR TITLE
Replace the gist links with imports.

### PR DESCRIPTION
**Pull request by jansiegel**
- Replace the gist links with imports (except for the localization demo, because of https://github.com/handsontable/hyperformula/pull/424)
- Modify the styling and structures for some of the demos.

### CODE REVIEW

File: advanced-usage/package.json [hunk 1]
```diff
@@ -8,6 +8,9 @@
     "start": "parcel index.html --open",
     "build": "parcel build index.html"
   },
+  "dependencies": {
+    "hyperformula": "0.1.0"
```
Comment (wojciechczerniak) [inline advanced-usage/package.json]: If, we're going to support whole `0.1.x` family of releases with those examples shouldn't we have some kind of wildcard here?

**Review by wojciechczerniak — CHANGES_REQUESTED** [review]
Looks good! But I have a question about hardcoded version number

File: advanced-usage/package.json (hunk 1 above)
Comment (jansiegel) [reply advanced-usage/package.json]: True!

**Review by wojciechczerniak — APPROVED** [review]
🏆

**Pull request merged**